scipy/stats/mstats_basic.py

```diff
@@ -548,7 +589,9 @@ def kendalltau(x, y, use_ties=True, use_missing=False, method='auto'):
         Defines which method is used to calculate the p-value [1]_.
         'asymptotic' uses a normal approximation valid for large samples.
         'exact' computes the exact p-value, but can only be used if no ties
-        are present. 'auto' is the default and selects the appropriate
+        are present. For sample size > 171, the exact computation may incur
+        roundoff error and can take very long.
```

Comment:
Roundoff error may occur for `n < 171` also.

Perhaps just state as:

"As n increases, the 'exact' computation time may grow and the result may lose some precision."

Reply:
Alright, changed (also in stats.py).